nephio-ansible-install/README.md

```diff
@@ -38,11 +38,13 @@ all:
     cloud_user: <username that is used to access the VM>
     github_username: <GitHub username>
     github_token: <GitHub personal access token>
-    github_organization: <optional, if a GitHub organization is used this should be filled out with the GitHub organization>
+    github_uri: github.com <replace this if you have a different github domain name in case of organization hosted github>
+    github_organization: <if a GitHub organization is used this should be filled out with the GitHub organization, else provide your github username>
     gitea_username: <gitea username>
     gitea_password: <gitea password>
     dockerhub_username: <dockerhub username>
     dockerhub_token: <dockerhub token or password>
+    validate_certs: true <change this to false if you want to avoid ssl/tls check>
```

Comment:
This is been used in ansible's many pre requisite tasks to avoid ssl/tls checks if set to false.